napari/utils/events/containers/_nested_list.py

```diff
@@ -205,7 +206,7 @@ def __setitem__(
         self, key: slice, value: Iterable[_T]
     ): ...  # pragma: no cover
 
-    def __setitem__(self, key: MaybeNestedIndex, value):
+    def __setitem__(self, key: MaybeNestedIndex, value) -> None:
```

Comment:
why value do not have type annotation if overloaded version exists?

Reply:
I thought overloaded implementations were meant to have no typing (https://docs.python.org/3/library/typing.html#overload), so I've removed the typing from it entirely and finished the typing in the overloaded signatures above.